app/Models/Device.php

```diff
@@ -251,10 +251,14 @@ public function canAccess($user)
             ->where('device_id', $this->device_id)->exists();
     }
 
-    public function formatUptime($short = false)
+    public function formatUptime($short = false, $downtime = false)
```

Comment:
Instead of a function that does two things, how about two functions that do one thing :)

I know I wrote it, but I don't really like the formatUptime. It is in the wrong place.  Probably move it to the Time util class as Time::formatInterval(CarbonInterval $interval) (or it could just accept seconds)

Then the above would be something like:
```php
'uptime' => Time::formatInterval($device->status ? $device->uptime : ($now - $device->last_polled)),
```

Reply:
For compatibility issue, the function Time::formatInterval uses seconds only.